minfraud/validation.py

```diff
@@ -246,10 +244,9 @@ def _credit_card_token(s: str) -> str:
     raise ValueError
 
 
-def _rfc3339_datetime(s: str) -> str:
-    if validate_rfc3339(s):
-        return s
-    raise ValueError
+_rfc3339_datetime = Match(
+    r"\A\d{4}-\d{2}-\d{2}[Tt]\d{2}:\d{2}:\d{2}(\.\d+)?(?:[Zz]|[+-]\d{2}:\d{2})\Z"
```

Comment:
Would we perhaps want to use `[0-9]` instead of `\d`? I'm wondering if we could incorrectly accept more digit characters than expected. Perhaps it is unlikely to arise in practice.

Reply:
Good thought. I added the ASCII flag instead.

This obviously will still allow invalid timestamps as it only validates the structure, not the content. However, I think that is sufficient for this purpose.